tests/options/test_data_drift.py

```diff
@@ -54,6 +54,15 @@ def _another_stattest():
     pass
 
 
+_features = dict([("cat1", "cat"), ("cat2", "cat"), ("num1", "num"), ("num2", "num")])
+
+
+def _features_st(*feature_func):
```

Comment:
I think it is too complicate code for tests - it is hard to read test cases with it

Can you rewrite it with explicit dicts in parametrize or at least add docs and comments for the method?

Reply:
Simplified test a bit.